tests/ChatTests/Profiles.hs

```diff
@@ -63,6 +63,10 @@ chatProfileTests = do
   describe "contact aliases" $ do
     it "set contact alias" testSetAlias
     it "set connection alias" testSetConnectionAlias
+  describe "pending connection users" $ do
+    it "change user id for pending connection" testUpdatePendingConnectionUserId
+    it "change user id for pending connection multiple times" testUpdatePendingConnectionUserIdMulti
+    it "can't change user id for non pending connections" testCantUpdateNonPendingConnectionUserId
```

Comment:
\+ test when connection is created as incognito, then switched to new user (should unset incognito)
\+ test when switched to new user, then to incognito (should be incognito in that user)
\+ test when switched to new user, then to incognito, then to non incognito (should stay non incognito in that new user)

maybe last one is a bit overkill.. better safe than sorry I guess